packages/core/index.ts

```diff
@@ -5,6 +5,7 @@ export type { PuckAction } from "./reducer/actions";
 export * from "./types/Config";
 export * from "./components/Button";
 export { Drawer } from "./components/Drawer";
+export * from './components/InputOrGroup';
```

Comment:
This change was added to from a "custom" field type you can use the built in input fields and maintain your own state internally before funneling the values up the tree. Ie I'd like to import `InputOrGroup` into my custom field, and render my own NumberField or CheckboxField for example.